service/policy/db/migrations/20250429000000_keys_table.sql

```diff
@@ -2,16 +2,16 @@
 -- +goose StatementBegin
 CREATE TABLE IF NOT EXISTS
```

Comment:
I think there might be unintended side effects of updating an existing migration that consumers of the latest platform will already have run and have present in their goose migrations table. I think it would be safer to capture the `CASCADE` statement clauses added in a separate migration.

Reply:
Right now it's largely up to the implementing developer to not mess up a migration, so any automated migration testing would be a really nice improvement.